docs/iceberg-rest-service.md

```diff
@@ -48,6 +48,17 @@ Deploy the Gravitino server to the `GRAVITINO_HOME` directory. You can find the
 The filter in the customFilters should be a standard javax servlet Filter.
 Filter parameters can also be specified in the configuration, by setting config entries of the form `gravitino.auxService.iceberg-rest.<class name of filter>.param.<param name>=<value>`
 
+### Iceberg metrics store configuration
+
+Gravitino provides a pluggable metrics store to save and delete Iceberg metrics.
+
+| Configuration item                                         | Description                                                                                                                                                                       | Default value | Required | Since Version |
+|------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------|----------|---------------|
+| `gravitino.auxService.iceberg-rest.metricsStore`           | The storage class name, designed to support the storage of Iceberg metrics, is pluggable and should extend from `com.datastrato.gravitino.catalog.lakehouse.iceberg.web.metrics`. | (none)        | No       | 0.4.0         |
+| `gravitino.auxService.iceberg-rest.metricsStoreRetainDays` | The retain days of Iceberg metrics, the value not greater than 0 means retain forever.                                                                                            | -1            | No       | 0.4.0         |
```

Comment:
The days to retain Iceberg metrics in store, the value smaller than 0 means retaining forever.